src/include/parser/postgresparser.h

```diff
@@ -78,106 +95,105 @@ class PostgresParser {
    * @param root list of parsed nodes
    * @return SQLStatementList corresponding to the parsed node list
    */
-  static std::vector<std::unique_ptr<parser::SQLStatement>> ListTransform(List *root);
+  static void ListTransform(ParseResult *parse_result, List *root);
```

Comment:
Add documentation above for @param ParseResult

Reply:
Done, also removed the @return